modules/oe_theme_helper/tests/src/Functional/Plugin/field_group/FieldInPageNavigationFormatterTest.php

```diff
@@ -0,0 +1,284 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme_helper\Functional\Plugin\field_group;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\field_group\Functional\FieldGroupTestTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test the in page navigation field group formatter.
```

Comment:
Tests the in-page navigation field group formatter.

Reply:
Changed